mkdocs.yml

```diff
@@ -46,4 +46,13 @@ theme:
   logo: assets/logo.png
   name: material
   palette:
-    scheme: cleura
+    - media: "(prefers-color-scheme: light)"
+      scheme: default
```

Comment:
nit: default may make us think it's a default color-scheme, where the extra_css in fact overrides that.
Would it make sense to call it cleura-light-mode and cleura-dark-mode ? For me, it looks more clear for marketing and other contributions.

Reply:
No, this is actually me having misunderstood how color schemes are named in mkdocs-material, initially. The light scheme is properly called `default`, and the dark scheme is called `slate`. What you can then do is override individual colors in either scheme. 

If you create a new scheme it inherits from `default`. So if there's a theme called `cleura` the two colors from `default` are easy to override, but if you want to have `cleura-dark` that's actually `slate` with a few tweaks, you have to reimplement all colors from `slate` in your dark scheme. 

The toggle is now implemented as suggested in [the documentation](https://squidfunk.github.io/mkdocs-material/setup/changing-the-colors/#color-palette-toggle), so as to follow [POLA](https://en.wikipedia.org/wiki/Principle_of_least_astonishment).

Reply:
I guess I misunderstood https://squidfunk.github.io/mkdocs-material/setup/changing-the-colors/#color-scheme . 
Thanks for clarifications.